src/set.cr

```diff
@@ -110,32 +195,48 @@ struct Set(T)
     end
   end
 
+  # Writes a string representation of the set to `io`
   def to_s(io)
     io << "Set{"
     join ", ", io, &.inspect(io)
     io << "}"
   end
 
+  # Returns `true` if the set is a subset of the `other` set
+  #
+  # This set must have fewer elements than the `other` set, and all of elements
+  # in this set must be present in the `other` set.
+  #
+  #     Set.new([1,5]).subset? Set.new([1,3,5]) # => true
   def subset?(other : Set)
```

Comment:
Maybe we should rename this to `strict_subset?` and add a `subset?` that allows equality? https://en.wikipedia.org/wiki/Subset